tools/OWNERS

```diff
@@ -10,5 +10,9 @@
         {name: 'ampproject/wg-performance'},
       ],
     },
+    {
+      pattern: 'experiment-config.js',
+      owners: [{name: 'ampproject/working-groups'}],
+    },
```

Comment:
@rcebulko Is this section okay here? Or should it be moved to `tools/experiments/OWNERS`?

Reply:
Ohhh! Good call. So I noticed that the `tools` directory only has an OWNERS file and the `experiments` directory within it. Should the OWNERS file be moved to `tools/experiments` or should the pattern here actually be `experiments/experiment-config.js` since I want to explicitly match this file only?

Reply:
Aha. If there are no other files under `tools/`, it makes sense to move the `OWNERS` file to `tools/experiments/`.

Reply:
I'll do that.

Reply:
Done